services/ui-src/src/utils/other/time.ts

```diff
@@ -22,6 +22,21 @@ export const calculateTimeByType = (timeType?: string): TimeShape => {
   return timeMap?.[timeType as keyof typeof timeMap] || noon;
 };
 
+/*
+ * Returns local time in HH:mm format with the "am/pm" indicator
+ * ex: 12:02pm
+ */
+export const getLocalHourMinuteTime = () => {
```

Comment:
is this value presented to the user? I've been trying to figure out how the function is used.

Reply:
helps if I test locally, not on main. I figured it out.